src/dali/plugin/input/rest/kraken.py

```diff
@@ -218,18 +212,18 @@ def load(self) -> List[AbstractTransaction]:
     def _compute_tx_set(self, trade_history, ledger, processed_transactions) -> List[AbstractTransaction]:
         result: List[AbstractTransaction] = []
 
-        transactions = processed_transactions[_IN] + processed_transactions[_OUT] + processed_transactions[_INTRA]
+        transactions: List[AbstractTransaction] = processed_transactions[_IN] + \
+                                                  processed_transactions[_OUT] + \
+                                                  processed_transactions[_INTRA]
         for key in transactions:
-            record = ledger[key]
-            timestamp_value: datetime = datetime.fromtimestamp(float(record[_TIMESTAMP]))
-            timestamp_value = timestamp_value.astimezone(self.__timezone)
-            timestamp_value = self.__timezone.normalize(timestamp_value)
+            record: Dict[str, str] = ledger[key]
```

Comment:
Maybe native data log debug goes here? Either way, but it's important to add it.

Reply:
Yea I'll add it here (~line 219). After looking at my code I'm not sure if I really needed to demarcate in/out/intra tx like I did lines 182-209. It just helped me think things through but maybe unimportant now. Anyways just putting it out there for scrutiny - I'll simplify it if there are strong feelings to clean it up further.